wrapanapi/systems/container/rhopenshift.py

```diff
@@ -690,6 +688,21 @@ def create_service(self, namespace, **kwargs):
         self.wait_service_exist(namespace=namespace, name=service_name)
         return output
 
+    def create_endpoints(self, namespace, **kwargs):
+        """Creates Endpoints entity using REST API.
+
+        Args:
+            namespace: openshift namespace where entity has to be created
+            kwargs: Endpoints data
+        Return: data if entity was created w/o errors
+        """
+        endpoints = self.kclient.V1Endpoints(**kwargs)
+        endpoints_name = endpoints.to_dict()['metadata']['name']
+        self.logger.info("creating endpoints %s", endpoints_name)
+        output = self.k_api.create_namespaced_endpoints(namespace=namespace, body=endpoints)
```

Comment:
Why not check if the output was successful before waiting for the endpoint to exist?

Reply:
output is different for different entities and I just can't certainly say what output is bad.
in addition, afaik it just raises exception if entity hasn't been created.